stanford/playbooks/deploy.yml

```diff
@@ -58,6 +60,20 @@
       with_items: "{{ COMMON_USER_INFO }}"
       tags:
         - users
+        - src
+    - name: Add remote for personal fork
+      command: "git remote add {{ item.name }} https://github.com/{{ item.name }}/configuration.git"
```

Comment:
Sadly, Ansible's `git` module doesn't appear to support multiple remotes,
so we have to do this manually.